app/presenters/speedy_af/proxy/media_object.rb

```diff
@@ -144,6 +144,10 @@ def sections_with_files(tag: '*')
     sections.select { |master_file| master_file.supplemental_files(tag: tag).present? }.map(&:id)
   end
 
+  def sections_with_rendering_files(tags)
+    tags.select { |t| sections_with_files(tag: t).present? }.present?
+  end
```

Comment:
The convention in Ruby is for methods returning boolean values to end with `?`. And there is an `any?` method which we could use instead of `select {}.present?` which should be more efficient.

```suggestion
  def sections_with_rendering_files?(tags)
    tags.any? { |t| sections_with_files(tag: t).present? }
  end
```

To keep parity between the proxy and the model, could you add this method to the MediaObject model as well? We should probably have modules shared between each Model and its Proxy (or at least MediaObject and MasterFile since we touch those most often) so that we only need to add methods in one place in cases like this. Something for @cjcolvar and I to talk over maybe.

Reply:
Thanks for the feedback! I will make the suggested changes.